dr/core/base/drpc.yaml

```diff
@@ -2,16 +2,14 @@
 apiVersion: ramendr.openshift.io/v1alpha1
 kind: DRPlacementControl
 metadata:
-  name: busybox-drpc
-  labels:
-    app: busybox-sample
+  name: drpc
 spec:
-  preferredCluster: "cluster1"
+  preferredCluster: cluster1
   drPolicyRef:
     name: dr-policy
   placementRef:
     kind: Placement
-    name: busybox-placement
+    name: placement
   pvcSelector:
     matchLabels:
       appname: busybox
```

Comment:
Label needs to be kustoimized in `dr/core/deployment-k8s-regional-rbd/kustomization.yaml` to `app: deployment-rbd`?

Reply:
Yes, the label is customized there.

Reply:
I am missing something here, the spec.pvcSelector should be customized in the file pointed to above, based on the commonLabels that is being set? I would assume the common labels is carried over to the actual workload artifacts as well (for e.g the `app` label). If not then this cannot (yet?) be customized here.

Reply:
pvcSelector is not customized - we customize appname in the kustomization.yaml to match the hardcoded value in the drpc.